src/wp-includes/block-patterns.php

```diff
@@ -158,11 +158,33 @@ function _register_core_block_patterns_and_categories() {
 	);
 }
 
+/**
+ * Normalize the pattern from the API (snake_case) to the format expected by `register_block_pattern` (camelCase).
```

Comment:
```suggestion
 * Normalize the pattern properties to camelCase.
 *
 * The API's format is snake_case, `register_block_pattern()` expects camelCase.
```

Reformatted into a short function description followed by a more descriptive explanation.